app/javascript/packages/countdown/countdown-alert-element.spec.ts

```diff
@@ -41,4 +46,20 @@ describe('CountdownAlertElement', () => {
       expect(element.show).to.have.been.called();
     });
   });
+
+  context('when time remaining has expired', () => {
+    it('redirects to the otp expired page', () => {
+      const element = createElement({
+        redirectURL: 'http://www.teapot.com',
```

Comment:
Minor: I'd suggest using a domain that we can trust (e.g. example.com, login.gov), since I don't think it's safe to assume what the contents of this URL may contain. In a similar note, I think it'd be good if we actually test that the redirect happened, which we could potentially do by assigning a hash fragment here.

Something like...

```ts
const element = createElement({
  redirectURL: '#teapot',
  // ..
});

// ...
await waitFor(() => window.location.hash === '#teapot');
```